app/main.ipynb

```diff
@@ -92,16 +129,34 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "df[\"item_price\"] = df[\"quantity\"] * df[\"price\"]\n",
+    "top_ten = df.sort_values(by=\"item_price\", ascending=False).groupby(\"name\").sum().head(10)\n",
+    "\n",
+    "plt.pie(top_ten[\"item_price\"], labels=top_ten.index, autopct=\"%1.1f%%\")\n",
+    "plt.title(\"Top 10 positions in menu by item price\")\n",
+    "plt.show()"
```

Comment:
this solution returns incorrect result. Are you sure that after group by you'll still have data sorted by quantity?